src/create-component.ts

```diff
@@ -0,0 +1,12 @@
+import { createComponent as createComponentOriginal, EventName } from '@lit-labs/react';
+
+export function createComponent(...args: Parameters<typeof createComponentOriginal>) {
+  // TODO: improve or remove the Polymer workaround
+  // 'createComponent' relies on key presence on the custom element class, but
+  // Polymer defines properties on the prototype when the first element
+  // is created. Workaround, create a throw-away element to make the prototype
+  // reliable for 'createComponent'.
+  const [, tagName] = args;
+  document.createElement(tagName);
```

Comment:
The set of properties can be read from `polymerElementClass._properties` even before the first element instance is created. For example:

```ts
const elementClass = args[2];
const mock = class {} as typeof elementClass;
Object.assign(mock.prototype, (elementClass as any)._properties);
args[2] = mock;
```

Reply:
Thanks for the tip!